processors/src/test/java/io/opentelemetry/contrib/eventbridge/EventToSpanEventBridgeTest.java

```diff
@@ -0,0 +1,161 @@
+                        spanEvent ->
+                            spanEvent
+                                .hasName("my.event-name")
+                                .hasTimestamp(100, TimeUnit.NANOSECONDS)
+                                .hasAttributesSatisfying(
+                                    attributes -> {
+                                      assertThat(attributes.get(stringKey("color")))
+                                          .isEqualTo("red");
+                                      assertThat(attributes.get(stringKey("shape")))
+                                          .isEqualTo("square");
+                                      assertThat(
+                                              attributes.get(
+                                                  longKey("log.record.observed_time_unix_nano")))
+                                          .isEqualTo(1000000L);
+                                      assertThat(
+                                              attributes.get(longKey("log.record.severity_number")))
+                                          .isEqualTo(Severity.DEBUG.getSeverityNumber());
+                                      assertThat(attributes.get(stringKey("log.record.body")))
+                                          .isEqualTo(
+                                              "{\"kvlistValue\":{\"values\":[{\"key\":\"number\",\"value\":{\"intValue\":\"1\"}},{\"key\":\"foo\",\"value\":{\"stringValue\":\"bar\"}},{\"key\":\"map\",\"value\":{\"kvlistValue\":{\"values\":[{\"key\":\"key\",\"value\":{\"stringValue\":\"value\"}}]}}}]}}");
+                                    })));
+  }
+
+  @Test
+  void noSpan_doesNotBridgeEvent() {
+    eventLogger
+        .builder("my.event-name")
+        .setTimestamp(100, TimeUnit.NANOSECONDS)
+        .setSeverity(Severity.DEBUG)
+        .put("foo", "bar")
+        .put("number", 1)
+        .put("map", Value.of(Collections.singletonMap("key", Value.of("value"))))
+        .setAttributes(Attributes.builder().put("color", "red").build())
+        .setAttributes(Attributes.builder().put("shape", "square").build())
+        .emit();
+
+    assertThat(spanExporter.getFinishedSpanItems()).isEmpty();
+  }
+
+  @Test
+  void nonRecordingSpan_doesNotBridgeEvent() {
+    Span span = tracer.spanBuilder("span").setSpanKind(SpanKind.INTERNAL).startSpan();
+    try (Scope unused = span.makeCurrent()) {
+      eventLogger
+          .builder("my.event-name")
+          .setTimestamp(100, TimeUnit.NANOSECONDS)
+          .setSeverity(Severity.DEBUG)
+          .put("foo", "bar")
+          .put("number", 1)
+          .put("map", Value.of(Collections.singletonMap("key", Value.of("value"))))
+          .setAttributes(Attributes.builder().put("color", "red").build())
+          .setAttributes(Attributes.builder().put("shape", "square").build())
+          .emit();
+    } finally {
+      span.end();
+    }
+
+    assertThat(spanExporter.getFinishedSpanItems()).isEmpty();
```

Comment:
It feels like the goal was to test the `isRecording()` check/functionality but instead it seems like this is just testing out the sampler. Maybe it's the case that the `SamplingResult` determines what the implementation of `isRecording()` returns? In any case, it's not super obvious to the reader why we would be testing that sampling works here.

Reply:
> Maybe it's the case that the SamplingResult determines what the implementation of isRecording() returns?

Yes that's right. I can add a comment to help the reader connect these dots.